lib/Analytics/Reporter.php

```diff
@@ -34,12 +34,14 @@ function getData() {
     $has_wc = $this->woocommerce_helper->isWooCommerceActive();
     $wc_customers_count = 0;
     if ($has_wc) {
-      $wc_customers_count = (int)Newsletter::rawQuery(
+      /** @var \stdClass */
+      $wc_customers = (int)Newsletter::rawQuery(
```

Comment:
The `(int)` cast should be removed.

Reply:
Hey, this is a great catch. I thought I removed it, but I guess I haven't.